blazor/treegrid/events.md

```diff
@@ -2337,5 +2337,227 @@ The [DetailDataBound](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Sy
     }
 }
 ```
+## BeforeCopyPaste
 
-N> We are not going to limit Tree Grid with these events, we will be adding new events in future based on the user requests. If the event, you are looking for is not in the list, then request [here](https://www.syncfusion.com/feedback/blazor-components).
+[BeforeCopyPaste]((https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.TreeGridEvents%601~BeforeCopyPaste.html) event will trigger before a cell is copied or pasted.
```

Comment:
use the above mentioned content here too

Reply:
Added the above content in events too.